app/jobs/regular/problem_check.rb

```diff
@@ -18,7 +18,12 @@ def execute(args)
       problems = check.call
       raise RetrySignal if problems.present? && retry_count < check.max_retries
 
-      problems.each { |problem| AdminDashboardData.add_found_scheduled_check_problem(problem) }
+      if problems.present?
+        problems.each { |problem| AdminDashboardData.add_found_scheduled_check_problem(problem) }
```

Comment:
This will use the new admin alerting system once it's in place.

Reply:
That's what I like to hear :fire: